.github/workflows/reusable-agent-build-container-images.yml

```diff
@@ -98,7 +100,7 @@ jobs:
           platforms: all
 
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@4c0219f9ac95b02789c1075625400b2acbff50b1 # v2.0.0
```

Comment:
As a best practice and for security reasons, we tend to pin all the non-official actions to specific commit sha (granted, with the official large organization github actions like Docker, potential security risk is smaller, but still).